waterdata/commands/lookup_generation/__init__.py

```diff
@@ -58,6 +98,7 @@ def generate_lookup_file(datadir, filename='nwis_lookup.json'):
         if 'params' in lookup_config:
             params.update(lookup_config.get('params'))
         resp = execute_get_request(CODE_HOST_ENDPOINT, path=lookup_config.get('urlpath'), params=params)
+        print(resp.url)
```

Comment:
Should probably remove the print statement

Reply:
Yep.